python/lsst/ts/dimm/dimm_csc.py

```diff
@@ -178,56 +229,78 @@ async def end_disable(self, id_data):
             Command ID and data
         """
 
-        await self.wait_loop(self.telemetry_loop_task)
+        try:
+            await self.wait_loop(self.telemetry_loop_task)
 
-        await self.wait_loop(self.seeing_loop_task)
+            await self.wait_loop(self.seeing_loop_task)
```

Comment:
Maybe break it down into two "try/except"? if the first one fails the second is never awaited.

Reply:
Done